bytestring.cabal

```diff
@@ -139,7 +139,7 @@ library
   -- * https://gitlab.haskell.org/ghc/ghc/-/issues/20525#note_385580
   -- * https://gitlab.haskell.org/ghc/ghc/-/issues/19417
```

Comment:
It would be good to update this comment. At least the first link suggests having the `gcc_s` extra-library.

Reply:
My reading of the issues comments correct leads me to conclude `gcc_s` is needed on Linux for template Haskell, but not on Windows because some things are "simulated"?